showcase/app/components/shw/label/index.gts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export interface ShwLabelSignature {
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLParagraphElement;
+}
+
+export default class ShwLabel extends Component<ShwLabelSignature> {
+  <template>
```

Comment:
I thought we needed the blocks interface in order to support `...attributes`? At least, I read that in a different PR. DYK?

Reply:
One thing that could be done is to use the `TemplateOnly` syntax; in which case it would look like this:
```
import type { TemplateOnlyComponent } from '@ember/component/template-only';
import { concat } from '@ember/helper';

export interface ShwDividerSignature {
  Args: {
    level?: 2;
  };
  Element: HTMLHRElement;
}

const ShwDivider: TemplateOnlyComponent<ShwDividerSignature> = <template>
  <hr class="shw-divider {{if @level (concat 'shw-divider--level-' @level)}}" />
</template>;

export default ShwDivider;
```

Reference: template-only component in HDS+: https://github.com/hashicorp/hds-plus/pull/29/files#diff-95b1b4bdc501dbdeb4e9adb3d3e258d7e3e84ec51428c87fbe91ffe75ba4d1ce

Reply:
I can do all of them in a batch, at the end of the conversion, if that simplifies things.

Reply:
I'm fine with either approach, I was using the examples given as my guide.

Reply:
Per discussion in the team sync, we'll merge this PR as-is and then @didoo will do some additional batch work at the end for template-only components.